src/main/java/Utils/messageHandler.java

```diff
@@ -57,19 +57,72 @@ public static void listShow(messageList list) {
             else {
                 doneSign = "";
             }
```

Comment:
Could the overuse of the word message and its variations - as variable name, class name, method name - get confusing for the writer and the reader? Maybe you could try using other names